tools/cloudflow-operator/src/main/scala/cloudflow/operator/action/runner/AkkaRunner.scala

```diff
@@ -0,0 +1,465 @@
+    new RoleBuilder()
+      .withNewMetadata()
+      .withName(Name.ofAkkaRole)
+      .withNamespace(namespace)
+      .withLabels(labels(Name.ofAkkaRole).asJava)
+      .withOwnerReferences(ownerReferences: _*)
+      .endMetadata()
+      .withKind("Role")
+      .withRules(createAkkaClusterPolicyRule, createEventPolicyRule)
+      .build()
+  }
+
+  private def akkaRoleBinding(
+      namespace: String,
+      role: Role,
+      labels: CloudflowLabels,
+      ownerReferences: List[OwnerReference]): RoleBinding = {
+    new RoleBindingBuilder()
+      .withNewMetadata()
+      .withName(Name.ofAkkaRoleBinding)
+      .withNamespace(namespace)
+      .withLabels(labels(Name.ofRoleBinding).asJava)
+      .withOwnerReferences(ownerReferences: _*)
+      .endMetadata()
+      .withKind("RoleBinding")
+      .withRoleRef(
+        new RoleRefBuilder()
+          .withApiGroup("rbac.authorization.k8s.io")
+          .withKind("Role")
+          .withName(role.getMetadata.getName)
+          .build())
+      .withSubjects(
+        new SubjectBuilder()
+          .withKind("ServiceAccount")
+          .withName(Name.ofServiceAccount)
+          .withNamespace(namespace)
+          .build())
+      .build()
+  }
+
+  private val createAkkaClusterPolicyRule = {
+    new PolicyRuleBuilder()
+      .withApiGroups("")
+      .withResources("pods")
+      .withVerbs("get", "list", "watch")
+      .build()
+  }
+
+  def prometheusConfig = PrometheusConfig(prometheusRules)
+
+  def resource(
+      deployment: App.Deployment,
+      app: App.Cr,
+      configSecret: Secret,
+      updateLabels: Map[String, String] = Map()): Deployment = {
+    // The runtimeConfig is already applied in the runner config secret, so it can be safely ignored.
+
+    val labels = CloudflowLabels(app)
+    val ownerReferences = List(AppOwnerReference(app.getMetadata.getName, app.getMetadata.getUid))
+    val appId = app.spec.appId
+    val podName = Name.ofPod(deployment.name)
+    val k8sStreamletPorts =
+      deployment.endpoint
+        .map { endpoint =>
+          endpoint.containerPort match {
+            case Some(cp) =>
+              Some(
+                new ContainerPortBuilder()
+                  .withContainerPort(cp)
+                  .withName(Name.ofContainerPort(cp))
+                  .build())
+            case _ => None
+          }
+        }
+        .flatten
+        .toList
+
+    val k8sPrometheusMetricsPort =
+      new ContainerPortBuilder()
+        .withContainerPort(PrometheusConfig.PrometheusJmxExporterPort)
+        .withName(Name.ofContainerPrometheusExporterPort)
+        .build()
+
+    val podsConfig = getPodsConfig(configSecret)
+
+    // TODO check if this is still valid.
+    // Pass this argument to the entry point script. The top level entry point will be a
+    // cloudflow-entrypoint.sh which will route to the appropriate entry point based on the
+    // arguments passed to it
+    val args = List("akka")
```

Comment:
I think this can be removed, the entrypoint is now already akka specific, so please remove comment and these args.